src/clj/witan/datasets.clj

```diff
@@ -0,0 +1,252 @@
+(ns witan.datasets
```

Comment:
Where has this been moved from?

Reply:
Workspaces api has some stats functions in that we needed to replicate.

Reply:
👍

Reply:
I wonder if these are also (or should be moved to) kixi.stats @otfrom ? https://github.com/MastodonC/kixi.stats